experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -691,6 +693,35 @@ describe('Node SDK', () => {
       });
     });
   });
+
+  describe('configure IdGenerator', async () => {
+    class CustomIdGenerator implements IdGenerator {
+      generateTraceId(): string {
+        return 'constant-test-trace-id';
+      }
+      generateSpanId(): string {
+        return 'constant-test-span-id';
+      }
+    }
+
+    it('should configure IdGenerator via config', async () => {
+      const idGenerator = new CustomIdGenerator();
+      const spanProcessor = new SimpleSpanProcessor(new ConsoleSpanExporter());
+      const sdk = new NodeSDK({
+        idGenerator,
+        spanProcessor,
+      });
+      sdk.start();
+
+      const span = trace.getTracer('test').startSpan('testName', {
```

Comment:
nit: is `'testName'` a good name for this span?

Reply:
i think it's fine. it's a span in a test that doesn't do anything